src/modules/card.js

```diff
@@ -0,0 +1,73 @@
+import {
+  popupMain, showNname, attSeasons, attPremiere, attGenres, attLanguage, popupPoster,
+  indexCard, ids, commentsList, popupNroComments,
+} from './elements.js';
+import { getSeassons, getComments } from './apis.js';
+import updateComments from './updateContent.js';
+import commentCounter from './commentsCounter.js';
+import { clickLike, displayLikes } from './likes.js';
+
+const showSeasons = getSeassons();
+
+const createCard = (show) => {
+  const cardContainer = document.createElement('div');
+  cardContainer.className = 'card';
+  cardContainer.id = show.id;
+  const cardImg = document.createElement('img');
+  cardImg.src = show.image.medium;
+  cardImg.alt = show.name;
+  const cardBody = document.createElement('div');
+  cardBody.className = 'card-body';
+  const showTitle = document.createElement('h3');
+  showTitle.innerText = show.name;
+  const likesDiv = document.createElement('div');
+  likesDiv.className = 'likes';
+  const showLikes = document.createElement('i');
+  showLikes.id = show.id;
+  showLikes.className = 'material-symbols-outlined';
+  showLikes.innerText = 'thumb_up';
+  const showLikesNumber = document.createElement('span');
+  showLikesNumber.className = 'likes_number';
+  showLikesNumber.innerText = '0';
+  clickLike(showLikes, showLikesNumber);
+  displayLikes(show.id, showLikesNumber);
+  cardContainer.appendChild(cardImg);
+  const cardCommentBtn = document.createElement('button');
+  cardCommentBtn.className = 'comments_btn';
+  cardCommentBtn.innerText = 'Comments';
+  cardContainer.appendChild(cardImg);
+  cardContainer.appendChild(cardBody);
+  cardBody.appendChild(showTitle);
+  cardBody.appendChild(likesDiv);
+  likesDiv.appendChild(showLikes);
+  likesDiv.appendChild(showLikesNumber);
+  cardContainer.appendChild(cardCommentBtn);
+  cardCommentBtn.addEventListener('click', () => {
```

Comment:
- [ Required ] I think it would be better if you fix the `comment` button to work as expected. I noticed that when I click on the comment button it failed to `add` comments to the specific movie item and the `counter` was also not updated. Please see the video below for more details on this and rectify it.

## Error on adding comments
https://github.com/Rebelzob/capstone-project-module2/assets/16392046/9efd89b3-4bed-4cd6-b048-094b2ffece84

Reply:
Thank you for the review